README.md

```diff
@@ -20,10 +20,10 @@ Website 🚀 <a href="https://contributte.org">contributte.org</a> | Contact 
 
 ## Usage
 
-To install latest version of `contributte/image-storage` use [Composer](https://getcomposer.com).
+To install latest version of `contributte/image-storage` use [Composer](https://getcomposer.org).
 
 ```
-composer install contributte/image-storage
+$ composer require contributte/image-storage
```

Comment:
Could you plese remove the `$` sign? That is not a part of the command

Reply:
what do you think @f3l1x  ?